pokemongo_bot/cell_workers/sleep_schedule.py

```diff
@@ -60,6 +61,8 @@ def _process_config(self):
             timedelta(
                 hours=duration_random_offset.hour, minutes=duration_random_offset.minute).total_seconds())
 
+        self.skip_current_sleep_cycle = self.config.get('skip_current_sleep_cycle', True)
```

Comment:
doh! skimmed over the `@@ -60,6 +61,8 @@ def _process_config(self):`  Sorry!  Carry on ;)